fuel-benches/src/lib.rs

```diff
@@ -1,23 +1,21 @@
-use std::{
-    io,
-    iter,
-};
-
+pub use fuel_core::database::Database;
 use fuel_core_interfaces::common::fuel_tx::{
     StorageSlot,
     TransactionBuilder,
 };
-use tempfile::TempDir;
-
-pub use fuel_core::database::Database;
 pub use fuel_core_interfaces::common::{
     consts::*,
     prelude::*,
 };
 pub use rand::Rng;
+use std::{
+    io,
+    iter,
+};
 
-fn new_db() -> io::Result<Database> {
-    TempDir::new().and_then(|t| Database::open(t.as_ref()).map_err(|e| e.into()))
+fn new_db() -> Database {
+    // when rocksdb is enabled, this creates a new db instance with a temporary path
```

Comment:
Should `fuel-core`'s rocksdb feature be enabled by default on this benchmark then? That way no benchmarks are accidentally done on much more performant RAM databases?